File: android/OhDormitory/app/src/main/java/kr/hs/emirim/uuuuri/ohdormitory/Model/SleepoutNotice.java
package kr.hs.emirim.uuuuri.ohdormitory.Model;

public class SleepoutNotice extends Notice {
    private String application_deadline;
    private String sleep_w_time;
    private String sleep_d_time;
    private int send;

    public SleepoutNotice(int notice_id, String title, String w_time, int type, String application_deadline, String sleep_w_time, String sleep_d_time, int send) {
        super(notice_id, title, w_time, type);
        this.application_deadline = application_deadline;
        this.sleep_w_time = sleep_w_time;
        this.sleep_d_time = sleep_d_time;
        this.send = send;
    }

    public String getApplication_deadline() {
        return application_deadline;
    }

    public String getSleep_w_time() {
        return sleep_w_time;
    }

    public String getSleep_d_time() {
        return sleep_d_time;
    }

    public int getSend() {
        return send;
    }
}

File: ios/OhDormitory/OhDormitory/SleepoutNotice.swift
//
//  SleepoutNotice.swift
//  OhDormitory
//
//  Created by 장명수 on 2018. 6. 11..
//  Copyright © 2018년 doori. All rights reserved.
//

import Foundation

class SleepoutNotice : Notice{
    /*
     "application_deadline": "2018-06-30",
     "sleep_w_time": "2018-06-30",
     "sleep_d_time": "2018-06-30",
     "send": "0"
     */
    var application_deadline : String
    var sleep_w_time : String
    var sleep_d_time: String
    var send : Int
    init(notice_id:Int, title:String, type:Int, w_time:String, d_time:String, application_deadline:String, sleep_w_time:String, sleep_d_time:String, send:Int){
        self.application_deadline = application_deadline
        self.sleep_w_time = sleep_w_time
        self.sleep_d_time = sleep_d_time
        self.send = send
        super.init(notice_id: notice_id, title: title, type: type, w_time: w_time, d_time: d_time)
    }
}
